src/styles/table.scss

```diff
@@ -137,6 +140,39 @@ $block: #{$fd-namespace}-table;
 
   /****** Elements ******/
 
+  &__empty {
+    &-table-message {
+      padding: 1rem;
+      text-align: center;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      align-content: center;
+      width: 100%;
+    }
+
+    &:not(.fixed-height) .fd-table__row:last-of-type {
+      .fd-table__cell {
+        border-bottom: 0;
+      }
+    }
+
+    .fd-table__body {
+      border-left: none;
+
+      @include fd-rtl() {
+        border-left: $fd-table-border;
+        border-left: $fd-table-border-var;
+        border-right: none;
+      }
+    }
+
+    &.fixed-height {
+      background-color: #fff;
+      background-color: var(--sapList_Background, #fff);
```

Comment:
First line is not needed, the second doesn't need the fallback. You can just have: background-color: var(--sapList_Background);

Reply:
done

Reply:
I think as it relates to the table's empty message item it shouldn't be a separate class, let it be modifier for base class, what do you think?

Reply:
Bump for this one, `.fd-fixed-height` not the correct class name.